Tests/UnitTests/Mocks/MockProductsManager.swift

```diff
@@ -31,6 +31,8 @@ class MockProductsManager: ProductsManager {
                 completion(result)
             }
         } else {
+            Logger.debug("\(type(of: self)): no stubbed products, returning fake products for \(identifiers)")
```

Comment:
This was confusing me, I had no idea where the products were coming from in a new test. It should help make it clear they're default stubs.